src/js/components/SidebarHeader.js

```diff
@@ -112,7 +112,7 @@ class SidebarHeader extends mixin(StoreMixin) {
         html: "Overview",
         id: "overview",
         onClick: () => {
-          SidebarActions.close();
+          SidebarActions.toggleSidebar(false);
```

Comment:
I am pretty confused by this API. I would not be sure that I really understand what it does from just reading `toggleSidebar(false)`. Does it set the sidebar to be closed always? Does it toggle the sidebar but without an animation? 

It seems to always set it to be closed, right?

I would really love to see this changed 😇